client/src/app/site/services/active-filters.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { StorageService } from 'src/app/gateways/storage.service';
+import { ActiveFiltersStoreService, OsFilter } from 'src/app/ui/base/filter-service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ActiveFiltersService implements ActiveFiltersStoreService {
+    constructor(private store: StorageService) {}
+
+    public save<V>(storageKey: string, filterDefinitions: OsFilter<V>[]): void {
+        this.store.set(`filter_` + storageKey, filterDefinitions);
```

Comment:
Do return this and change the return type to `Promise<void>` -> one can await for the "ready event".